Source/DafnyLanguageServer/Workspace/Notifications/VerificationDiagnosticsParams.cs

```diff
@@ -352,21 +352,6 @@ Position Position
       }
     }
 
-    // If the verification never starts on this node, it means there is nothing to verify about it.
-    // Returns true if a status was updated
-    public bool SetVerifiedIfPending() {
-      if (StatusCurrent == CurrentStatus.Obsolete) {
```

Comment:
Where did this code go?

Reply:
It's gone. Apparently it wasn't needed, or do you think there's a test missing?

Reply:
It was probably obsolete, I vaguely remember changing the way we set verified on methods without verification condition, and I might have forgotten to delete a method.